examples/test_passphrase.cpp

```diff
@@ -0,0 +1,53 @@
+#define BOOST_TEST_MODULE passphrase_test
+#include <boost/test/included/unit_test.hpp>
+#include <boost/algorithm/string.hpp>
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include <cli_widgets/hidden_input_guard.hpp>
+#include <boost/process.hpp>
+#include <boost/filesystem.hpp>
+
+ BOOST_AUTO_TEST_CASE(passphrase_test) {
+  namespace bp = boost::process;
+  namespace fs = boost::filesystem;
+
+  bp::opstream in;
+  bp::ipstream out;
+  bp::pipe p;
+  std::vector<std::string> outline;
+  std::string value;
+  std::string path_to_exe;
+
+  fs::path current_path = fs::absolute(fs::current_path());
+  for (fs::directory_entry &entry : fs::recursive_directory_iterator{current_path}) {
+    if (entry.path().filename().string() == "stdin_hide_guard" ||entry.path().filename().string() == "stdin_hide_guard.exe" ) {
+      path_to_exe = entry.path().string();
+    }
+  }
+
+  std::vector<std::string> many_password = {"Un p@ssw0rd balnéaire à Zürich ?"};
+
+  for (auto password : many_password){
+    bp::child c(path_to_exe,"automatic",bp::std_out > out, bp::std_in < in);
```

Comment:
automatic is the only possible password ?

Reply:
We should test that passswords with spaces are usable

Reply:
Automatic if for have an agrc ==1, he is not put as a password